src/modules/XMLParser.ts

```diff
@@ -14,16 +21,18 @@ class XMLParser {
     try {
       return (
         xml
-          // we will replace all non closing br tags, with self-closing once
-          .replace(/<br>/gm, '<br />')
-          .replace(/'(?=[^>]*<)/g, '&apos;')
-          // we need to restructure the link structure into one single link element
+          // replace all non closing br tags with self-closing once (fixed with CORE-13424)
+          .replace(/<br>/g, '<br />')
```

Comment:
If the /gm is removed, only occurences of br in the first line of the xml will be replaced by br/. 
If this the problem with non closing br is fixed in FS, why should we keep the code?

Reply:
Your understanding of the modifier `m` is not correct ;)
Why keep the code --> I wanted to keep the handling of data which may be still stored in the wrong format in CaaS...